pkg/services/rpcsrv/subscription_test.go

```diff
@@ -601,28 +602,40 @@ func TestWSClientsLimit(t *testing.T) {
 				cfg.ApplicationConfiguration.RPC.MaxWebSocketClients = limit
 			})
 
-			dialer := websocket.Dialer{HandshakeTimeout: 5 * time.Second}
+			dialer := websocket.Dialer{HandshakeTimeout: 10 * time.Second}
```

Comment:
Move it to a separate commit, attach the latest failing log. We need a reason for every change.

Reply:
Let's firstly see if it solves the problem.

Reply:
waiting for tests, in case 10s isn't enough.